app/services/queue_courtesy_email_service.rb

```diff
@@ -0,0 +1,51 @@
+class QueueCourtesyEmailService
+  def initialize(content_change_or_message)
+    raise ArgumentError.new("Must be a Message or a Content Change") unless
+        content_change_or_message.is_a?(Message) ||
+          content_change_or_message.is_a?(ContentChange)
```

Comment:
Minor but this guard clause seems rather unusually presented. Might be an easier read as:

if !content_change_or_message.is_a?(Message) && content_change_or_message.is_a?(ContentChange)
  raise ArgumentError.new("Must be a Message or a Content Change")
end